css/style.css

```diff
@@ -116,7 +116,9 @@ ul.nav {
 
 a, a:hover, a:focus {
   color: #6eceb2;
-  font-weight: bold;
+}
+a.inline-link {
```

Comment:
I like the white background better.  And I agree this is an improvement on the original, because the light green in Open Sans Light isn't strong enough in contrast. 

If you don't quite like this, what about the emphasized text from the Style guide, which would be: 

font-weight: bold;
font-style: italic;
color: #f4364c; // styleguide red
? 

Otherwise, this looks good to me.